### ISSUE TITLE
Failed to reenable joining

### ISSUE THREAD

**Issue by Kevlordofcode** [Koenkk/zigbee2mqtt#1601]
# Bug Report
## What happened

My network (mainly Aqara sensors) suddenly stopped working after several weeks without any problem. Nothing was posted on mqtt anymore.
To fix it I have tried a lots of things, including reseting the cc2531, reflashing it several times, deleting database.db etc.
It is still not working. Zigbee2mqtt runs fine but I cannot pair any device. I have noticed numerous strange entries in the logs:
`error: Failed to reenable joining`

I could not find any other issues mentioning this error so I figured out I would open a new one.
I thank you all in advance for your kind assistance!

## Debug Info
zigbee2mqtt version: 1.4.0
CC253X firmware version: 20190425
Debug log: https://pastebin.com/0WzhS0qZ

**Comment by Koenkk** [comment]
Could you post the log when running with `DEBUG=* npm start`?

**Comment by Kevlordofcode** [comment]
Thank you for your quick answer. Here is the requested log: https://pastebin.com/U1qPVPd0

**Comment by Koenkk** [comment]
Somehow the CC2531 doesn't respond, can you try with https://github.com/Koenkk/Z-Stack-firmware/raw/e98e0e8c2b5826ad4d29a1046bfeaf78c08b4c53/coordinator/max_stability/CC2531/CC2531_MAX_STABILITY_20190315.zip

**Comment by Kevlordofcode** [comment]
Done. Here is the debug log: https://pastebin.com/P4aksVng

Unfortunately it did not solve the problem.

It's like if the stick had become somehow corrupted and it cannot be completely reseted even by reflashing it, is it possible?

Please note that it sometimes manages to reenable the joining but it's only temporary:

```
2019-6-6 15:47:18 - debug: Saving state to file /opt/zigbee2mqtt/data/state.json
2019-6-6 15:47:24 - error: Failed to reenable joining
2019-6-6 15:50:41 - info: Succesfully reenabled joining
2019-6-6 15:52:18 - debug: Saving state to file /opt/zigbee2mqtt/data/state.json
2019-6-6 15:54:01 - info: Succesfully reenabled joining
2019-6-6 15:57:18 - debug: Saving state to file /opt/zigbee2mqtt/data/state.json
2019-6-6 15:57:24 - error: Failed to reenable joining
2019-6-6 16:00:44 - error: Failed to reenable joining
```

**Comment by Koenkk** [comment]
It's strange indeed, how do you flash the stick? (this should also provide an option to clear it).

**Comment by Kevlordofcode** [comment]
I use the CC debugger with cc-tools on MacOS, with the exact command I found in you documentation:
`sudo ./cc-tool -e -w CC2531ZNP-Prod.hex`

**Comment by Koenkk** [comment]
`-e` should erase the chip. To be sure, you could try on windows with flash programmer, if that doesn't help I think there is something wrong with the CC2531 stick.

**Comment by Kevlordofcode** [comment]
I'll try on Windows next week (if I find a suitable PC), otherwise I guess I will just order a new CC2531. Thanks anyway for the help!

**Comment by vrolland78** [comment]
Same issue "error: Failed to reenable joining" in the log and no more update.
Only 1 aqara temp sensor for test actually.
Only solution is to restart the service :(

**Comment by Kevlordofcode** [comment]
On my side I figured it out eventually. This was due to Homeassistant trying to access a zwave coordinator stick that was no longer there. I have disabled openzwave in Homeassistant and now it's working perfectly. I would suggest you check if something can somehow interfere with your usb port and disable it, it might help!

**Comment by rippi46** [comment]
Hello,
since the update from zigbee2mqtt to version 1.5.1 I have the same problem with my CC2530 via ESP8266. After a restart of the service zigbee2mqtt everything works fine, but after some time I get the following message:

`Jul 23 23:29:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:29:29 PM Failed to reenable joining
Jul 23 23:32:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:32:09 PM Failed to reenable joining
Jul 23 23:34:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:34:49 PM Failed to reenable joining
Jul 23 23:37:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:37:29 PM Failed to reenable joining
Jul 23 23:40:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:40:09 PM Failed to reenable joining
Jul 23 23:42:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:42:49 PM Failed to reenable joining
Jul 23 23:45:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:45:29 PM Failed to reenable joining
Jul 23 23:48:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:48:09 PM Failed to reenable joining
Jul 23 23:50:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:50:49 PM Failed to reenable joining
Jul 23 23:53:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:53:29 PM Failed to reenable joining
Jul 23 23:56:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:56:09 PM Failed to reenable joining
Jul 23 23:58:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-23 11:58:49 PM Failed to reenable joining
Jul 24 00:01:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:01:29 AM Failed to reenable joining
Jul 24 00:04:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:04:09 AM Failed to reenable joining
Jul 24 00:06:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:06:49 AM Failed to reenable joining
Jul 24 00:09:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:09:29 AM Failed to reenable joining
Jul 24 00:12:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:12:09 AM Failed to reenable joining
Jul 24 00:14:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:14:49 AM Failed to reenable joining
Jul 24 00:17:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:17:29 AM Failed to reenable joining
Jul 24 00:20:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:20:09 AM Failed to reenable joining
Jul 24 00:22:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:22:49 AM Failed to reenable joining
Jul 24 00:25:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:25:29 AM Failed to reenable joining
Jul 24 00:28:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:28:09 AM Failed to reenable joining
Jul 24 00:30:49 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:30:49 AM Failed to reenable joining
Jul 24 00:33:29 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:33:29 AM Failed to reenable joining
Jul 24 00:36:09 fhem-pi npm[31386]:   zigbee2mqtt:error 2019-7-24 12:36:09 AM Failed to reenable joining
`
With a second stick, which is connected via USB I do not have this problem, although the same firmware is on both sticks.

This is how the output looks after starting the service.

`Jul 24 09:05:11 fhem-pi systemd[1]: Started zigbee2mqtt2.
Jul 24 09:05:13 fhem-pi npm[30892]: > zigbee2mqtt@1.5.1 start /opt/zigbee2mqtt2
Jul 24 09:05:13 fhem-pi npm[30892]: > node index.js
Jul 24 09:05:16 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:16 AM Logging to directory: '/opt/zigbee2mqtt2/data/log/2019-07-24.09-05-15'
Jul 24 09:05:17 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:17 AM Starting zigbee2mqtt version 1.5.1 (commit #ac3b924)
Jul 24 09:05:17 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:17 AM Starting zigbee-shepherd
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM zigbee-shepherd started
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM Coordinator firmware version: '20190223'
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM Currently 22 devices are joined:
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002f6a6ab (0x00158d0002f6a6ab): MLI-404011 - Müller Licht Tint remote control (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x90fd9ffffedd7c9e (0x90fd9ffffedd7c9e): LED1545G12 - IKEA TRADFRI LED bulb E26/E27 980 lumen, dimmable, white spectrum, opal white (Router)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0001ad45dd (0x00158d0001ad45dd): RTCGQ01LM - Xiaomi MiJia human body movement sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0001b980b8 (0x00158d0001b980b8): WSDCGQ11LM - Xiaomi Aqara temperature, humidity and pressure sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x000d6ffffe1d3ace (0x000d6ffffe1d3ace): E1524 - IKEA TRADFRI remote control (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d00031d719e (0x00158d00031d719e): 404006/404008/404004 - Müller Licht Tint LED bulb GU10/E14/E27 350/470/806 lumen, dimmable, opal white (Router)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0xd0cf5efffe1aea7a (0xd0cf5efffe1aea7a): E1524 - IKEA TRADFRI remote control (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002b3188c (0x00158d0002b3188c): DJT11LM - Xiaomi Aqara vibration sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002d701b8 (0x00158d0002d701b8): MCCGQ01LM - Xiaomi MiJia door & window contact sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002b932ce (0x00158d0002b932ce): RTCGQ11LM - Xiaomi Aqara human body movement and illuminance sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d00019cba2e (0x00158d00019cba2e): WSDCGQ01LM - Xiaomi MiJia temperature & humidity sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d00019fbf39 (0x00158d00019fbf39): MCCGQ01LM - Xiaomi MiJia door & window contact sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d00029c08a6 (0x00158d00029c08a6): RTCGQ11LM - Xiaomi Aqara human body movement and illuminance sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0001b9253b (0x00158d0001b9253b): WSDCGQ11LM - Xiaomi Aqara temperature, humidity and pressure sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d00027cefec (0x00158d00027cefec): MFKZQ01LM - Xiaomi Mi/Aqara smart home cube (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002b74df6 (0x00158d0002b74df6): MCCGQ01LM - Xiaomi MiJia door & window contact sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002b74d76 (0x00158d0002b74d76): MCCGQ01LM - Xiaomi MiJia door & window contact sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0003248906 (0x00158d0003248906): 404000/404005/404012 - Müller Licht Tint LED bulb GU10/E14/E27 350/470/806 lumen, dimmable, color, opal white (Router)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0002b23b0c (0x00158d0002b23b0c): DJT11LM - Xiaomi Aqara vibration sensor (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d0001e80850 (0x00158d0001e80850): unkown - undefined unknown (EndDevice)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0x00158d000327ee70 (0x00158d000327ee70): 404006/404008/404004 - Müller Licht Tint LED bulb GU10/E14/E27 350/470/806 lumen, dimmable, opal white (Router)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM 0xd0cf5efffe1ae577 (0xd0cf5efffe1ae577): LED1537R6 - IKEA TRADFRI LED bulb GU10 400 lumen, dimmable, white spectrum (Router)
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:05:21 AM `permit_join` set to  `true` in configuration.yaml.
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:05:21 AM Allowing new devices to join.
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:05:21 AM Set `permit_join` to `false` once you joined all devices.
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM Zigbee: allowing new devices to join.
Jul 24 09:05:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:21 AM Connecting to MQTT server at mqtt://localhost:1884
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM zigbee-shepherd ready
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM Connected to MQTT server
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/bridge/state', payload 'online'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002f6a6ab', payload '{"linkquality":0}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x90fd9ffffedd7c9e', payload '{"state":"OFF","linkquality":23,"brightness":49,"color_temp":454,"color_mode":2,"color":{"x":0.46,"y":0.411}}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0001ad45dd', payload '{"occupancy":false,"linkquality":86,"battery":100,"voltage":3025}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0001b980b8', payload '{"temperature":28.53,"linkquality":70,"humidity":44.67,"pressure":998,"battery":45,"voltage":2905}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x000d6ffffe1d3ace', payload '{"linkquality":21,"battery":74}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00031d719e', payload '{"state":"OFF","brightness":254,"color_temp":370,"color":{"x":0.381,"y":0.377},"linkquality":47,"color_mode":2}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0xd0cf5efffe1aea7a', payload '{"linkquality":28,"battery":34}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b3188c', payload '{"linkquality":73,"angle":5,"angle_x":3,"angle_y":-84,"angle_z":5,"angle_x_absolute":87,"angle_y_absolute":174,"battery":100,"voltage":3025}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002d701b8', payload '{"contact":true,"linkquality":57,"battery":100,"voltage":3025}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b932ce', payload '{"illuminance":7,"linkquality":78,"occupancy":false,"battery":97,"voltage":2995}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00019cba2e', payload '{"temperature":23.76,"linkquality":52,"humidity":60.37,"battery":100,"voltage":3025}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00019fbf39', payload '{"contact":true,"linkquality":139,"battery":97,"voltage":2995}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00029c08a6', payload '{"illuminance":40,"linkquality":44,"occupancy":false,"battery":100,"voltage":3075}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0001b9253b', payload '{"temperature":28.2,"linkquality":73,"humidity":47.03,"pressure":992.7,"battery":62,"voltage":2935}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00027cefec', payload '{"battery":68,"voltage":2945,"linkquality":89,"angle":-7.29,"side":1,"from_side":5,"to_side":1}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b74df6', payload '{"contact":true,"linkquality":65,"battery":100,"voltage":3015}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b74d76', payload '{"contact":true,"linkquality":68,"battery":100,"voltage":3025}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0003248906', payload '{"state":"ON","brightness":254,"color":{"x":0.458,"y":0.41,"saturation":100,"hue":240},"color_temp":370,"color_mode":2,"linkquality":0}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b23b0c', payload '{"angle":178,"linkquality":39,"angle_x":-1,"angle_y":0,"angle_z":-89,"angle_x_absolute":91,"angle_y_absolute":90,"battery":97,"voltage":2995}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d000327ee70', payload '{"state":"OFF","brightness":45,"color":{"x":0.381,"y":0.377},"color_temp":370,"linkquality":0,"color_mode":2}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/0xd0cf5efffe1ae577', payload '{"color_temp":370,"color_mode":2,"color":{"x":0.46,"y":0.411},"linkquality":0,"state":"ON","brightness":254}'
Jul 24 09:05:23 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:05:23 AM MQTT publish: topic 'zigbee2mqtt2/bridge/config', payload '{"version":"1.5.1","commit":"ac3b924","coordinator":20190223,"log_level":"info","permit_join":true}'
Jul 24 09:06:13 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:06:13 AM Failed to configure 0x000d6ffffe1d3ace (0x000d6ffffe1d3ace) ('Error: Timed out after 10000 ms') (attempt #1)
Jul 24 09:06:13 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:06:13 AM This can be ignored if the device is working properly
Jul 24 09:07:03 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:07:03 AM Failed to configure 0xd0cf5efffe1aea7a (0xd0cf5efffe1aea7a) ('Error: Timed out after 10000 ms') (attempt #1)
Jul 24 09:07:03 fhem-pi npm[30892]:   zigbee2mqtt:warn 2019-7-24 9:07:03 AM This can be ignored if the device is working properly
Jul 24 09:08:02 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:08:02 AM Successfully reenabled joining
Jul 24 09:08:48 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:08:48 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00027cefec', payload '{"battery":80,"voltage":2965,"linkquality":86,"angle":-7.29,"side":1,"from_side":5,"to_side":1}'
Jul 24 09:10:42 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:10:42 AM Successfully reenabled joining
Jul 24 09:13:22 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:13:22 AM Successfully reenabled joining
Jul 24 09:14:21 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:14:21 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002d701b8', payload '{"contact":true,"linkquality":63,"battery":100,"voltage":3025}'
Jul 24 09:16:02 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:16:02 AM Successfully reenabled joining
Jul 24 09:18:42 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:18:42 AM Successfully reenabled joining
Jul 24 09:21:22 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:21:22 AM Successfully reenabled joining
Jul 24 09:23:29 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:23:29 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d00019fbf39', payload '{"contact":true,"linkquality":139,"battery":97,"voltage":2995}'
Jul 24 09:24:02 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:24:02 AM Successfully reenabled joining
Jul 24 09:26:42 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:26:42 AM Successfully reenabled joining
Jul 24 09:29:22 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:29:22 AM Successfully reenabled joining
Jul 24 09:32:02 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:32:02 AM Successfully reenabled joining
Jul 24 09:34:42 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:34:42 AM Successfully reenabled joining
Jul 24 09:35:53 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:35:53 AM MQTT publish: topic 'zigbee2mqtt2/0x00158d0002b3188c', payload '{"linkquality":81,"angle":5,"angle_x":3,"angle_y":-84,"angle_z":5,"angle_x_absolute":87,"angle_y_absolute":174,"battery":100,"voltage":3025}'
Jul 24 09:37:22 fhem-pi npm[30892]:   zigbee2mqtt:info 2019-7-24 9:37:22 AM Successfully reenabled joining
`

Any idea?

Greeting rippi46

**Comment by petjek** [comment]
Same here. Everything worked fine for a while. I already added 6 devices (Xiaomi and Ikea) While adding the sixth device the "Failed to reenable joining"-message came up. Service crashed and I tried to resart. It took a while after "sudo systemctl stop zigbee2mqtt.service" until I got back to the prompt and from there on it is not even possible to start the service again.
When starting manual I see this:
`
arne@FHEM:/opt/zigbee2mqtt$  npm start 
> zigbee2mqtt@1.5.1 start /opt/zigbee2mqtt
> node index.js
  zigbee2mqtt:info 8/20/2019, 8:27:29 PM Logging to directory: '/opt/zigbee2mqtt/data/log/2019-08-20.20-27-28'
  zigbee2mqtt:info 8/20/2019, 8:27:29 PM Starting zigbee2mqtt version 1.5.1 (commit #ac3b924)
  zigbee2mqtt:info 8/20/2019, 8:27:29 PM Starting zigbee-shepherd
  zigbee2mqtt:info 8/20/2019, 8:27:41 PM Error while starting zigbee-shepherd, attempting to fix... (takes 60 seconds) (Error: request timeout)
  zigbee2mqtt:info 8/20/2019, 8:28:41 PM Starting zigbee-shepherd
  zigbee2mqtt:error 8/20/2019, 8:29:20 PM Error while starting zigbee-shepherd! (Error: request timeout)
  zigbee2mqtt:error 8/20/2019, 8:29:20 PM Press the reset button on the stick (the one closest to the USB) and start again
  zigbee2mqtt:error 8/20/2019, 8:29:20 PM Failed to start
        {"message":"request timeout","stack":"Error: request timeout\n    at CcZnp.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/cc-znp/index.js:208:22)\n    at Object.onceWrapper (events.js:286:20)\n    at CcZnp.emit (events.js:198:13)\n    at Timeout.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/cc-znp/index.js:192:18)\n    at ontimeout (timers.js:436:11)\n    at tryOnTimeout (timers.js:300:5)\n    at listOnTimeout (timers.js:263:5)\n    at Timer.processTimers (timers.js:223:10)"}
  zigbee2mqtt:error 8/20/2019, 8:29:20 PM Exiting...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.5.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.5.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/arne/.npm/_logs/2019-08-20T18_29_57_094Z-debug.log
arne@FHEM:/opt/zigbee2mqtt$
 `

With sudo systemctl start zigbee2mqtt.service and sudo journalctl -u zigbee2mqtt.service -f I get the following:

`
arne@FHEM:/opt/zigbee2mqtt$ sudo systemctl start zigbee2mqtt.service
arne@FHEM:/opt/zigbee2mqtt$ sudo journalctl -u zigbee2mqtt.service -f
-- Logs begin at Mon 2019-06-10 08:51:26 CEST. --
Aug 20 20:33:07 FHEM systemd[1]: zigbee2mqtt.service: Scheduled restart job, restart counter is at 20.
Aug 20 20:33:07 FHEM systemd[1]: Stopped zigbee2mqtt.
Aug 20 20:33:07 FHEM systemd[1]: zigbee2mqtt.service: Failed to reset devices.list: Operation not permitted
Aug 20 20:33:07 FHEM systemd[1]: Started zigbee2mqtt.
Aug 20 20:33:08 FHEM npm[3702]: > zigbee2mqtt@1.5.1 start /opt/zigbee2mqtt
Aug 20 20:33:08 FHEM npm[3702]: > node index.js
Aug 20 20:33:08 FHEM npm[3702]:   zigbee2mqtt:info 8/20/2019, 8:33:08 PM Logging to directory: '/opt/zigbee2mqtt/data/log/2019-08-20.20-33-08'
Aug 20 20:33:08 FHEM npm[3702]:   zigbee2mqtt:info 8/20/2019, 8:33:08 PM Starting zigbee2mqtt version 1.5.1 (commit #ac3b924)
Aug 20 20:33:08 FHEM npm[3702]:   zigbee2mqtt:info 8/20/2019, 8:33:08 PM Starting zigbee-shepherd
Aug 20 20:33:20 FHEM npm[3702]:   zigbee2mqtt:info 8/20/2019, 8:33:20 PM Error while starting zigbee-shepherd, attempting to fix... (takes 60 seconds) (Error: request timeout)
`

Had this problem two times now and the only way to solve it the first time was to completely remove the installation and start from scratch.
